explorer/src/lib/containers/statistics-panel/StatisticsPanel.svelte

```diff
@@ -43,7 +43,9 @@
 
   onNetworkChange((network) => {
     marketDataStore.getData(network);
+    nodeLocationsStore.reset();
```

Comment:
Is a reset really needed here? If yes why on the node locations but not on the market data?

Reply:
Yes, it is needed. I do not reset the market data as that api endpoint is not dependent on the node.